lib/reversetunnel/local_cluster.go

```diff
@@ -411,7 +411,7 @@ func (s *localCluster) dialAndForward(params reversetunnelclient.DialParams) (_
 	ctx := s.srv.ctx
 
 	if params.GetUserAgent == nil && !params.TargetServer.IsOpenSSHNode() {
-		return nil, trace.BadParameter("agentless node require an agent getter")
+		return nil, trace.BadParameter("user agent getter is required for teleport nodes")
```

Comment:
Is this always a bug? If so, here and in the other always-bug errors:
```suggestion
		return nil, trace.BadParameter("user agent getter is required for teleport nodes (this is a bug)")
```